libs/core/src/lib/panel/panel.component.ts

```diff
@@ -106,14 +96,10 @@ export class PanelComponent implements CssClassBuilder, OnChanges, OnInit, OnDes
         return this._elementRef;
     }
 
-    /** @hidden */
-    private _listenOnExpandedChange(): void {
-        this._subscription = this._panelService.expanded$
-            .pipe(filter(value => value.isExpandTriggerClick))
-            .subscribe(value => {
-                this.expanded = value.isExpanded;
-                this.expandedChange.emit(this.expanded);
-                this._cdRef.detectChanges();
-            });
+    /** Methods that toggles the Panel Content */
+    toggleExpand(): void {
+        this.expanded = !this.expanded;
+        this.expandedChange.emit(this.expanded);
+        this._cdRef.detectChanges();
```

Comment:
There is no need to manually force detect changes, it will be triggered with `click` event.

Reply:
Good point, removed it